repl/src/main/scala/ammonite/repl/Util.scala

```diff
@@ -184,7 +184,7 @@ object SyntaxError{
   def msg(code: String, p: fastparse.core.Parser[_], idx: Int) = {
     val locationString = {
       val (first, last) = code.splitAt(idx)
-      val lastSnippet = last.split('\n')(0)
+      val lastSnippet = last.split('\n').headOption getOrElse ""
```

Comment:
can we use `.getOrElse("")` here? I tend to not use infix notation except for cases where we're chaining lots of things together